lib/takeaway.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'current_order'
+require 'twilio-ruby'
+
+class Takeaway
+
+  attr_reader :menu
+
```

Comment:
How could you make the Takeaway more flexible?  I.e. How could you allow for different menus?